src/nat/middleware/red_teaming/red_teaming_middleware.py

```
@@ -0,0 +1,342 @@
+        matches = jsonpath_expr.find(value_to_modify)
+        if len(matches) == 0:
+            raise ValueError(f"No matches found for target_field: {self._target_field} in value: {value}")
+        if len(matches) > 1:
+            matches = self._resolve_multiple_field_matches(matches)
+        else:
+            matches = [matches[0]]
+        modified_values = [
+            self._apply_payload_to_simple_type(match.value, self._attack_payload, self._payload_placement)
+            for match in matches
+        ]
+        for match, modified_value in zip(matches, modified_values):
```

Ruff (0.14.8)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)